Resources/public/javascript/forms.js

```diff
@@ -1,16 +1,61 @@
+let removePopup = `<div class="sonata-ba-delete popup-file-delete mfp-hide">
```

Comment:
This is not good yet, right now dutch translations are directly in this file. 
What happends when we create a CMS for English people?

Also, maybe a bit weird to have all this HTML in the JS file?